docs/opensap/index.md

```diff
@@ -63,17 +61,193 @@ To inspect the source code of a particular state, either checkout the correspond
 | [course/4_3_advanced_services_solution](https://github.com/SAP/cloud-s4-sdk-book/tree/course/4_3_advanced_services_solution) | 4/3 | Solution | [ZIP](https://github.com/SAP/cloud-s4-sdk-book/archive/course/4_3_advanced_services_solution.zip) |
 | [course/4_4_eventing](https://github.com/SAP/cloud-s4-sdk-book/tree/course/4_4_eventing) | 4/4 | Example for consuming messages | [ZIP](https://github.com/SAP/cloud-s4-sdk-book/archive/course/4_4_eventing.zip) |
 
-## Errata
+## Example Code and Additional Material
+
+This section describes where to find the example code for the specific units. It also links additional material and important commands for each unit, where applicable.
+
+### Week 1 Introducing Intelligent ERP
+
+#### Unit 1 to 4
+
+Just enjoy watching the videos for units 1 to 4. There is no preparation required.
+
+#### Unit 5 Building a Hello World Extension App
+
+Before starting with his unit make sure you fulfilled the requirements listed above.
+
+In the openSAP course, you are asked to create a new project from our archetypes.
+The command to generate a new project is the following.
+
+For bash:
+
+``` bash
+mvn archetype:generate \
+"-DarchetypeGroupId=com.sap.cloud.s4hana.archetypes" \
+"-DarchetypeArtifactId=scp-cf-tomee" \
+"-DarchetypeVersion=2.1.2" \
+"-DgroupId=com.sap.cloud.s4hana.examples" \
+"-DartifactId=address-manager" \
+"-Dversion=1.0-SNAPSHOT"
+```
+
+For cmd (Windows):
+``` cmd
+mvn archetype:generate ^
+"-DarchetypeGroupId=com.sap.cloud.s4hana.archetypes" ^
+"-DarchetypeArtifactId=scp-cf-tomee" ^
+"-DarchetypeVersion=2.1.2" ^
+"-DgroupId=com.sap.cloud.s4hana.examples" ^
+"-DartifactId=address-manager" ^
+"-Dversion=1.0-SNAPSHOT"
+```
+
+The .gitignore file referenced in unit can be found [here](http://tiny.cc/s4sdk-gitignore).
+
+##### Troubleshooting:
+* When starting the application locally, make sure to run the command `mvn tomee:run` in the application folder of your project. Otherwise, the command might fail with an error message like: 
+`No plugin found for prefix 'tomee'`.
+
+* If your subacount on SCP could not be created, please do the following:
```

Comment:
I doubt that SCP newbies will find this an easy task. Does that really happen? If so, can we maybe show what to do, maybe an animated gif screencast or so..

Reply:
I hope that it does not occur so often anymore, as we reported it. Creating a video is too much effort for that issue IMO.

Reply:
ok